tests/utils/dockercli/volumelifecycle.go

```diff
@@ -90,21 +90,8 @@ func ReadFromVolume(ip, volName, containerName, fileName string) (string, error)
 
 // DeleteVolume helper deletes the created volume as per passed volume name.
 func DeleteVolume(ip, name string) (string, error) {
-	log.Printf("Destroying volume [%s]\n", name)
-	var out string
-	var err error
-
-	for attempt := 0; attempt < maxRemoveVolAttempt; attempt++ {
-		out, err = ssh.InvokeCommand(ip, dockercli.RemoveVolume+name)
-		if err != nil && strings.Contains(out, pluginInitError) {
-			misc.SleepForSec(waitTime)
-			log.Printf("Volume cannot be deleted yet as plugin initialization still in progress. Retrying...")
-			continue
-		} else {
-			break
-		}
-	}
-	return out, err
+	log.Printf("Deleting volume [%s]\n", name)
```

Comment:
I think this retrial was added due to a design a design decision that volume cannot be deleted in phase where plugin initialization is in progress.

//CC @ashahi1  @shuklanirdesh82 


Since there isn't a specific reason in PR description as to why this change is needed.. can you point to the necessary reason?

If I am not wrong, did you imply test case should handle volume deletion(or prior step to it) depending on test scenario and retry again. 

As far as I know, doing this change will intermittently fail VM listener test case.